controllers/forum-controllers.js

```diff
@@ -8,3 +8,14 @@ exports.getCategories = (request, response, next) => {
       next(err);
     });
 };
+
+exports.getReviewById = (request, response, next)=>{
+  const reviewId = request.params.review_id;
+
+  selectReviewById(reviewId)
+    .then((reviewObj)=>{
+      response.status(200).send(reviewObj)
```

Comment:
a couple of points about reviewObj:
1) this is not an object, as we see in the test this is an array. 
2) it's not a common practice to add the type to a variable like this unless you need to differentiate. It tends to be more useful to stick with informative variable naming 
3) be sure to stick to the convention of sending responses back on an object eg. { review } or { review: review }